package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "devDependencies": {
+        "laravel-mix": "^6.0.41"
+    },
+    "workspaces": {
+        "packages": [
+            "modules/backend",
+            "modules/system"
+        ]
+    }
+}
```

Comment:
```suggestion
}

```

Reply:
This file shouldn't be in core. I'll remove it.